Assets/Scripts/Views/AssetViewer.cs

```diff
@@ -100,4 +118,55 @@ public void Lighten(GameObject prefab)
     {
         prefab.GetComponentInChildren<Image>().color = Color.white;
     }
+
+    public void MoveTo(GameObject prefab, Vector2 target, float speed, MovementTypes style)
+    {
+        moveCoroutine = StartCoroutine(MoveAsset(prefab, target, speed, style));
+    }
+    
+    //For SmoothDamp only; higher speed = slower movement
+    private IEnumerator MoveAsset(GameObject prefab, Vector2 target, float speed, MovementTypes style)
+    {
+        isMoving = true;
+        movingPrefab = prefab;
+        targetPosition = target;
+        
+        Vector2 currentPosition = prefab.transform.position;
+        Vector2 velocity = Vector2.zero;
+
+        speed *= Time.deltaTime;
+
+        while (Vector2.Distance(currentPosition, target) > .1f)
```

Comment:
If you change this to `while (Mathf.RoundToInt(Vector2.Distance(currentPosition, target)) > 0)`, it shaves about a second off in testing with barely any visible difference.

Reply:
It also looks much better than having the arbitrary .1f, I'll include it.